demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php

```diff
@@ -445,20 +445,22 @@ protected function getStatementStatusData(
         }
 
         // save status counts
-        $esResultMeta = $statementQueryResult->getFilterSet();
-        $aggregations = $esResultMeta['filters'];
+        $aggregations = $statementQueryResult->getFilterSet()['filters'];
         foreach ($aggregations[StatementService::AGGREGATION_STATEMENT_STATUS] as $aggregationBucket) {
-            $statusValue = $aggregationBucket['value'];
-            $statusCount = $aggregationBucket['count'];
-            $statementStatusData[$statusValue]['count'] = $statusCount;
-
-            // add link with filterhash to assessment table
-            if (0 < $statusCount) {
-                $statementStatusData[$statusValue]['url'] = $this->generateAssessmentTableFilterLinkFromStatus(
-                    $statusValue,
-                    $procedureId,
-                    'statement'
-                );
+            if (array_key_exists($aggregationBucket['value'], $statementStatuses)) {
+                $statusValue = $aggregationBucket['value'];
+                $statusCount = $aggregationBucket['count'];
+                $statementStatusData[$statusValue]['count'] = $statusCount;
+                $statementStatusData[$statusValue]['freq'] = $priorityInitialValues;
```

Comment:
Are you sure you need to set the priorityInitialValues here. I think this happens already in line 432. There they are added for each status. And in the if statement they will be added for the same statuses again. When I remove this line, the result seems the same.

Reply:
You're right. thx.
09788e334349618283248317725d663d569dd288